### PR TITLE
Fix website build

### PR DESCRIPTION
**Pull request by SenseException**
Pushing these changes into `staging` should allow to test them.

### CODE REVIEW

**Comment by SenseException** [comment]
I've put the branch temporarily on https://staging.doctrine-project.org to test the changes.

**Comment by malarzm** [comment]
FWIW the missing docs in the sidebar on https://staging.doctrine-project.org/projects/doctrine-mongodb-odm/en/2.2/index.html are present 🎉

**Comment by SenseException** [comment]
@malarzm Yep, I've rebased my branch before I created the PR. :-)

**Review by greg0ire — APPROVED** [review]

**Comment by malarzm** [comment]
Let's ship it? 🐑

**Comment by SenseException** [comment]
Sheeped :sheep:

File: .github/workflows/deployment.yml [hunk 1]
```diff
@@ -53,11 +53,15 @@ jobs:
       with:
         composer-options: "--no-progress --no-suggest --no-dev"
 
-    - name: "apt-get install yarn"
-      run: "sudo apt-get update && sudo apt-get install -y yarn"
+    - name: "Installation of node"
+      uses: "actions/setup-node@v2"
+      with:
+        node-version: '10.22.0'
```
Comment (stof) [inline .github/workflows/deployment.yml:59]: why node 10 and not an uptodate version ? Node 10 is EOL

**Pull request merged**